From 04f29a111a2e1f9372c743f47bda42ae76979f2b Mon Sep 17 00:00:00 2001
From: francescotescari <36578133+francescotescari@users.noreply.github.com>
Date: Tue, 28 Jan 2020 16:27:03 +0100
Subject: [PATCH 02/36] fix XssFetchProductDataPhp1Min

---
 php_action/fetchProductData.php | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/php_action/fetchProductData.php b/php_action/fetchProductData.php
index b584a3a..92aa057 100644
--- a/php_action/fetchProductData.php
+++ b/php_action/fetchProductData.php
@@ -6,6 +6,9 @@ $sql = "SELECT product_id, product_name FROM product WHERE status = 1 AND active
 $result = mysqli_query($conn, $sql);
 
 $data = mysqli_fetch_all($result);
+foreach ($data as $k => $v){
+	$data[$k][1] = htmlentities($v[1]);
+}
 
 mysqli_close($conn);
 
-- 
2.7.4

