From cfbd1aa01b96d1506f5a29a3c0b34c8102650a93 Mon Sep 17 00:00:00 2001
From: francescotescari <36578133+francescotescari@users.noreply.github.com>
Date: Tue, 28 Jan 2020 17:51:11 +0100
Subject: [PATCH 20/36] fix XssOrdersPhp46Min

---
 orders.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/orders.php b/orders.php
index fbe649f..625bd43 100644
--- a/orders.php
+++ b/orders.php
@@ -422,7 +422,7 @@ if($_GET['o'] == 'add') {
 				  <div class="form-group">
 				    <label for="grandTotal" class="col-sm-3 control-label">Grand Total</label>
 				    <div class="col-sm-9">
-				      <input type="text" class="form-control" id="grandTotal" name="grandTotal" disabled="true" value="<?php echo $data[8] ?>"  />
+				      <input type="text" class="form-control" id="grandTotal" name="grandTotal" disabled="true" value="<?php echo htmlentities($data[8]) ?>"  />
 				      <input type="hidden" class="form-control" id="grandTotalValue" name="grandTotalValue" value="<?php echo $data[8] ?>"  />
 				    </div>
 				  </div> <!--/form-group-->	
-- 
2.7.4

