From f26f852686bcee461d9a18c6af29a62329e83aca Mon Sep 17 00:00:00 2001
From: francescotescari <36578133+francescotescari@users.noreply.github.com>
Date: Tue, 28 Jan 2020 16:58:33 +0100
Subject: [PATCH 05/47] fix XssOrdersPhp11Min

---
 orders.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/orders.php b/orders.php
index 8732fa3..e20aff8 100644
--- a/orders.php
+++ b/orders.php
@@ -108,7 +108,7 @@ if($_GET['o'] == 'add') {
 			  							$productData = mysqli_query($conn, $productSql);
 
 			  							while($row = mysqli_fetch_array($productData)) {									 		
-			  								echo "<option value='".$row['product_id']."' id='changeProduct".$row['product_id']."'>".$row['product_name']."</option>";
+			  								echo "<option value='".$row['product_id']."' id='changeProduct".$row['product_id']."'>".htmlentities($row['product_name'])."</option>";
 										 	} // /while 
 
 			  						?>
-- 
2.19.2.windows.1

