From 44a5868e78addf800037ad60ff6f9066baaa69ec Mon Sep 17 00:00:00 2001
From: francescotescari <36578133+francescotescari@users.noreply.github.com>
Date: Tue, 28 Jan 2020 17:36:57 +0100
Subject: [PATCH 10/47] fix XssOrdersPhp31Min

---
 orders.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/orders.php b/orders.php
index ea12525..e712321 100644
--- a/orders.php
+++ b/orders.php
@@ -351,7 +351,7 @@ if($_GET['o'] == 'add') {
 			  				</td>
 			  				<td style="padding-left:20px;">			  					
 			  					<input type="text" name="rate[]" id="rate<?php echo $x; ?>" autocomplete="off" disabled="true" class="form-control" value="<?php echo htmlentities($orderItemData['rate']); ?>" />			  					
-			  					<input type="hidden" name="rateValue[]" id="rateValue<?php echo $x; ?>" autocomplete="off" class="form-control" value="<?php echo $orderItemData['rate']; ?>" />			  					
+			  					<input type="hidden" name="rateValue[]" id="rateValue<?php echo $x; ?>" autocomplete="off" class="form-control" value="<?php echo htmlentities($orderItemData['rate']); ?>" />			  					
 			  				</td>
 							<td style="padding-left:20px;">
 			  					<div class="form-group">
-- 
2.19.2.windows.1

